yarn-project/noir-contracts/src/contracts/test_contract/src/utils.nr

```diff
@@ -0,0 +1,50 @@
+use dep::aztec::hash::{sha256_to_field};
+
+// Computes a content hash of a deposit/mint_public message.
+// Refer TokenPortal.sol for reference on L1.
```

Comment:
This is a little funky. What benefit is there of having the same being reused for a test with the `test_contract` when testing it separately?

Reply:
I couldn't call the appropriate methods in tokenbridge because the simulator would enqueue several other function calls across the token contract and the token_bridge contract and I would have to mock all of them, but they would emit their own nullifiers thus failing the point of the test.

Reply:
Also for the utils, could you not just import it?

Reply:
ah yea i could create a library for it